### DISCUSSION TITLE
[Question] Copy PVC backup to another Cluster

### DISCUSSION THREAD

**Discussion by pratapagiri** [vmware-tanzu/velero#3330] [Community support Q&A] [answered]
We have requirement to take backup of existing Cassandra PVC (and probably creating tar.gz/zip of the backup), download and move it to another environment. Looked at Velero and it's very promising. However, I'm not clear about how do I create zip file (or tar.gz), so that I can move it different environment and restore it there.
Can someone help me out on how to do this?

**Comment by zubron** [comment, marked as answer]
Hi @pratapagiri! Please see our documentation on [Cluster Migration](https://velero.io/docs/v1.5/migration-case/). It provides details on how to configure your two environments to use the same backup storage location so that any backups from one cluster can be used to restore the other cluster. Velero will handle the creation and reading of any archive file. Given that you want to migrate PVC data, you may need to [enable restic](https://velero.io/docs/v1.5/restic/) to handle that.
